src/ExportConverters.mss

```diff
@@ -641,7 +610,7 @@ function ConvertNoteStyle (style) {
     {
         case (CrossNoteStyle)
         {
-            noteStyle = 'cross';
+            noteStyle = 'x';
```

Comment:
I think we should only assign `'x'` if `noteobj.ParentNoteRest.Duration <=256`, otherwise `'diamond'`